character-controller.js

```diff
@@ -491,6 +497,21 @@ class PlayerBase extends THREE.Object3D {
       _emitEvents();
     }
   }
+  setTarget(target) { // set both head and eyeball target;
+    if (target) {
+      this.headTarget.copy(target);
+      this.headTargetInverted = true;
+      this.headTargetEnabled = true;
+
+      this.eyeballTarget.copy(target);
+      this.needLimitEyeballTargetRange = true;

```

Comment:
I don't get the point of this. The range needs to always be clipped to not have the eyes too far our.

Reply:
Can we just get rid of this?

Reply:
@avaer Ok, deleted `needLimitEyeballTargetRange`.

I added `needLimitEyeballTargetRange` mainly due to two concerns:
1. Prevent re-calculate root-to-head angle, use `lastNeedsHeadTarget` which already calculated in head for eyeball.
2. To keep the same behavior as before, which eyeball will forced to rotate toward mirror, regardless to angle limit.
Before:
![image](https://user-images.githubusercontent.com/10785634/170744479-2de02063-bf72-4d53-b14f-d8f49959a22d.png)
![image](https://user-images.githubusercontent.com/10785634/170744499-e151a86a-a637-45e0-8974-40a1111970db.png)

For 1, though it simplified calculations, but really made logics complicated and unstable.
For 2, I think it's fine to not rotate eyeballs in front of mirror if angle is too large. ( Still look into mirror if angle is in limit )
After:
![image](https://user-images.githubusercontent.com/10785634/170745971-7434392c-22f3-43a4-9643-3f784eb48edf.png)
![image](https://user-images.githubusercontent.com/10785634/170746073-59f42116-92de-4f58-85f8-4820aec5074e.png)
![image](https://user-images.githubusercontent.com/10785634/170746153-b2400171-4437-4fff-a744-f920538314c9.png)

So deleted `needLimitEyeballTargetRange`.
Re-calculated `needEyeballTarget` instead of using `lastNeedsHeadTarget`, untangled logics, and can set different max angle limitation  if need.

BTW, fixed `mouse look` feature not work problem.